harness/tests/export_presets.cfg

```diff
@@ -49,7 +49,7 @@ custom_features=""
 export_filter="all_resources"
 include_filter="*.txt"
 exclude_filter=""
-export_path=""
+export_path="../tests-win64-export/Godot Kotlin Tests.apk"
```

Comment:
This line is quite specific for your setup no?

Reply:
oups, did not wanted to add it